src/runtime/internal/Component.ts

```diff
@@ -43,6 +43,16 @@ export function mount_component(component, target, anchor) {
 			run_all(new_on_destroy);
 		}
 		component.$$.on_mount = [];
+
+		if (fragment.bbl) {
```

Comment:
Unless I'm mistaken, this mechanism won't forward events with handlers attached after the component has been mounted.